### DISCUSSION TITLE
Python/%E7%B3%BB%E5%88%97%E6%96%87%E7%AB%A0/23%E7%A7%8D%E8%AE%BE%E8%AE%A1%E6%A8%A1%E5%BC%8F/

### DISCUSSION THREAD

**Discussion by giscus** [cpython666/cpython666.github.io#1] [Announcements]
# Python/%E7%B3%BB%E5%88%97%E6%96%87%E7%AB%A0/23%E7%A7%8D%E8%AE%BE%E8%AE%A1%E6%A8%A1%E5%BC%8F/

Python斗罗的代码笔记

http://localhost:8080/Python/%E7%B3%BB%E5%88%97%E6%96%87%E7%AB%A0/23%E7%A7%8D%E8%AE%BE%E8%AE%A1%E6%A8%A1%E5%BC%8F/

**Comment by cpython666** [comment]
泰酷啦!
